resources/network_virtual_networks.go

```diff
@@ -427,27 +416,58 @@ func fetchNetworkVirtualNetworks(ctx context.Context, meta schema.ClientMeta, pa
 	}
 	return nil
 }
-func fetchNetworkVirtualNetworkSubnets(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
-	vn := parent.Item.(network.VirtualNetwork)
+func resolveNetworkVirtualNetworksDhcpOptionsDnsServers(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
SDK v0.5.2 has a InetArray resolver I suggest maybe to use that?

Reply:
It seems that resolver does not handle strings. it returns `dhcp_options_dns_servers expected TypeInetArray got *string `